package.json

```diff
@@ -4,9 +4,9 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
-    "test:open": "npm run lint && cypress open",
-    "lint": "eslint ./cypress/e2e/**/*.ts",
-    "lint:fix": "npm run lint -- --fix"
+    ... scripts existentes, no borrar
+    "test": "cypress run"
+    ... scripts existentes, no borrar
```

Comment:
The change was not extactly this, maybe it is a little confussing but that where just comments to point out that the "scripts" section had already many scripts which shouldn''t be deleted.

The only change expected here is to add another script which is `"test": "cypress run"`, but let the other without changes